modules/core/src/lib/layer.js

```diff
@@ -324,6 +325,13 @@ export default class Layer extends Component {
         attributeManager.invalidateAll();
       }
     }
+
+    if (!this.isComposite && changeFlags.transitionsChanged) {
```

Comment:
Add some comment on how we treat (or not) composite layers, i.e. the `if (!this.isComposite)` part